onnxruntime/core/session/inference_session.cc

```diff
@@ -1053,13 +1054,21 @@ Status InferenceSession::Run(const RunOptions& run_options, const std::vector<st
   TraceLoggingWriteStart(ortrun_activity, "OrtRun");
 #endif
   Status retval = Status::OK();
-
+  const Env& env = Env::Default();
   try {
     if (!is_inited_) {
       LOGS(*session_logger_, ERROR) << "Session was not initialized";
       return Status(common::ONNXRUNTIME, common::FAIL, "Session not initialized.");
     }
 
+    // check the frequency to send Evalutaion Stop event
+    if (TimeDiffMicroSeconds(time_sent_last_evalutation_start_) > kDurationBetweenSendingEvaluationStart) {
```

Comment:
Does the time units match here?
The function name indicate the unit is ms, but it looks like the unit of kDurationBetweenSendingEvaluationStart is us.

Reply:
It is matching.

Reply:
I double checked it. It should be right. All is measured in microseconds(us):smile: 

---
In reply to: [360187176](https://github.com/microsoft/onnxruntime/pull/2704#discussion_r360187176) [](ancestors = 360187176)